src/utils/translations.js

```diff
@@ -0,0 +1,8 @@
+// Local translations when used outside of sense
```

Comment:
just did this for now to collect all the strings needed, so we can get all checked at the same time. Then we will have them both here and in the client, so that the client overrides these ones when in sense. In theory we only need the warnings outside of the client, but easier to have all of them and use the same setups.